msteams-platform/m365-apps/actions-in-m365.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/16/2023
+ms.author: mosdevdocs
+author: v-bvishnu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+>
+> * Actions for Microsoft 365 is available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+>
+> * Actions is supported for Microsoft 365 apps on web and desktop clients only and isn't supported on Outlook and Microsoft Teams.
+
+Actions enable your app to integrate seamlessly into the user's workflow and guide users to your app based on their intent and the content at hand, making task completion more efficient.
+Actions help you enhance your app's visibility and engagement with minimal development effort and reduce the need to switch contexts across various Microsoft 365 apps.
+
+The following graphic is an example of open page Action where the user intends to view the list of suppliers. In Microsoft 365, the user right-clicks on the excel file and selects **Related suppliers** to view the list of suppliers for the Northwind company. The Northwind page opens with the list of suppliers. The user saves time by checking the result with a single click without opening the app and the Excel file each time manually.
+
+  :::image type="content" source="images/actions-in-m365-app.gif" alt-text="Graphic shows the user right-clicks on the excel file and select the related suppliers.":::
+
+## Understand how Actions work
+
+Actions are the combination of intent, object, and handler. Actions represent the task that the user wants to perform where intent is the user’s desired action, object is the function to be executed, and handler is the way to perform the action on the object.
```

Comment:
This paragraph does not describe intent, object and handler in the right way. Please refer to my doc:

From a developer's perspective, an Action is built using a combination of Intent + Object + Handler. When a user intends to accomplish a task, it can be represented as intent + object, with the intent denoting the verb describing the user's desired action, and the object representing the noun on which the action is to be performed.    

As a developer, your role is to receive the user's intent + object input and construct the corresponding handler that facilitates task completion for the users.   

To build an Action, you will define the intent, object, and handler of your actions in the manifest. And in your handler, use Teams JS V2 to receive the Action information to create a seamless user experience for performing users specific tasks.  